openquakeplatform/openquakeplatform/static/js/irv_viewer_d3_tree.js

```diff
@@ -511,9 +546,23 @@
             nodeEnter.append("text")
                 .attr("class", "pointer")
                 .style("fill", "#0000EE")
-                .attr("x", function(d) { return "-1em"; })
+                .attr("x", function(d) {
+                    if ( d.field == 'SVI') {
```

Comment:
you should do here as in the other PR, using the type instead of the name or field. Otherwise you might have that d.field is something like 'SVI_2' and it wouldn't work.